python/lsst/meas/algorithms/__init__.py

```diff
@@ -23,7 +23,26 @@
 """lsst.meas.algorithms
 """
 from __future__ import absolute_import
-from .algorithmsLib import *
+
+import lsst.afw.image
+import lsst.afw.math
+from lsst.afw.image import DefectBase
```

Comment:
Do we actually refer to `lsst.meas.algorithms.DefectBase`?  That might be a mistake we want to fix.